### PR TITLE
Remove bazel shutdown on automation/test.sh

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

dockerized bazel shutdown was failing on several builds:

https://search.ci.kubevirt.io/?search=Usage%3A+bazel+%3Ccommand%3E+%3Coptions%3E&maxAge=336h&context=3&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dhiller** [comment]
FYI @ormergi

**Review by enp0s3 — APPROVED** [review]
/lgtm
/approve

**Comment by enp0s3** [comment]
/hold
lets see whats wrong with the lanes

File: automation/test.sh [hunk 1]
```diff
@@ -226,7 +226,7 @@ kubectl get nodes
 
 make cluster-sync
 
-hack/dockerized bazel shutdown
+hack/dockerized "bazel shutdown"
```
[the remark this answers is not in the document]
Comment (rmohr) [reply automation/test.sh:229]: Seems to be no longer necessary :+1:

[the remark this answers is not in the document]
Comment (dhiller) [reply automation/test.sh:229]: ack, then I'm just going to remove it

[the remark this answers is not in the document]
Comment (rmohr) [reply automation/test.sh]: The purpose of it originally was to save memory during the functest run. The functests run pretty long and the bazel server keeps running in the background, still consuming memory.

**Pull request merged**